### PR TITLE
Pager fix

### PR DESCRIPTION
**Pull request by jforissier**
Fixing the issues identified after adding `$(platform-cflags)` to the compiler flags for `version.o`.

### CODE REVIEW

File: core/arch/arm/kernel/link.mk [hunk 1]
```diff
@@ -58,14 +58,16 @@ objs-init-rem += core/arch/arm/tee/arch_svc.o
 objs-init-rem += core/arch/arm/tee/arch_svc_asm.o
 objs-init-rem += core/arch/arm/tee/init.o
 objs-init-rem += core/arch/arm/tee/entry.o
-entries-init += _start
+entries-init += _start core_v_str
```
Comment (jenswi-linaro) [inline core/arch/arm/kernel/link.mk]: Is it necessary to add `core_v_str` here?

Comment (jforissier) [inline core/arch/arm/kernel/link.mk]: Actually, no. I'll remove it.

**Comment by pascal-brand-st-dev** [comment]
Reviewed-by: Pascal Brand <pascal.brand@linaro.org>
    Tested-by: Pascal Brand <pascal.brand@linaro.org> (STM platform)

**Comment by jenswi-linaro** [comment]
```
Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>
```

**Comment by jforissier** [comment]
Squashed and review tags added.

**Pull request merged**